elements/pl-integer-input/pl-integer-input.mustache

```diff
@@ -79,3 +97,20 @@ Scientific notation is not accepted.
 {{#shortformat}}
 integer
 {{/shortformat}}
+
+{{#format_error}}
+    <p> Invalid format:
+        {{#format_error_message}}
+            {{{ format_error_message }}}
+        {{/format_error_message}}
+        {{^format_error_message}}
+            the submitted answer could not be interpreted as an integer.  No symbolic input, fractions, or scientific notation are allowed.
+        {{/format_error_message}}
+    </p>
+    <p> Example <i>valid</i> inputs: <code class="user-output">5</code> &nbsp; <code class="user-output">-17</code> </p>
+    <p> Example <i>invalid</i> inputs: <code class="user-output">6+9</code> &nbsp; <code class="user-output">3.7</code> &nbsp; <code class="user-output">3*pi</code> &nbsp; <code class="user-output">6e19</code> </p>
+    <hr>
+    <small>
+        <p> The submitted answer must be formatted as a signed 64-bit integer. These numbers lie roughly in the range of -9 &times; 10<sup>19</sup> to 9 &times; 10<sup>19</sup>.</p>
```

Comment:
Is this actually true? It looks like Python will parse any number of digits, store it in JSON, and parse it back afterwards. So I think this is actually unlimited precision?

Reply:
<img width="391" alt="image" src="https://user-images.githubusercontent.com/1790491/75054335-7f8e0f00-5498-11ea-96dd-2c4e65012eb6.png">

Just tried it now and am getting this error.  From what I see on the documentation you are right though, so not sure why it is breaking.

Reply:
Looks like this is actually failing when `np.isfinite(a_sub_parsed)` is called on the parsed integer value.  This is probably safe to remove since it doesn't make much sense here anyway?

Reply:
I guess `np.isfinite()` is coercing the value to double precision before it checks for Inf/NaN, so it makes sense that a large integer would convert to Inf and then fail the check.

I agree that `np.isfinite()` seems unnecessary (and in fact harmful) for integers.